src/main/java/frc/robot/subsystem/Wrist.java

```diff
@@ -0,0 +1,51 @@
+package frc.robot.subsystem;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+
+import edu.wpi.first.math.controller.PIDController;
+
+public class Wrist {
+
+    enum WristStates {
+        HIGH,
+        LOW,
+        FLAT;
+    }
+
+    private CANSparkMax pivotMotor;
+    private WristStates state;
+    private PIDController controller;
+
+    public Wrist() {
+        state = WristStates.FLAT;
+        pivotMotor = new CANSparkMax(1, MotorType.kBrushless);
+
+        pivotMotor.getEncoder().setPosition(0);
+
+        controller = new PIDController(0, 0, 0);
+    }
+
+    public void setstate(WristStates state) {
+        this.state = state;
+    }
+
+    public void periodic() {
+        switch (state) {
+            case HIGH:
+                pivotMotor.set(controller.calculate(pivotMotor.getEncoder().getPosition(), 225));
```

Comment:
do you know what units this is in? nativley rev reports motor position in rotations iirc so you would be telling the motor to rotate 255 rotations. You can change what it reports with pivotMotor.getEncoder().setPositionConversionFactor(x) where x will be multipleid by whatever output it returns.

Reply:
add units to constants.java btw!